src/uct/ib/mlx5/dv/ib_mlx5_ifc.h

```diff
@@ -1251,7 +1252,9 @@ struct uct_ib_mlx5_qpc_bits {
     uint8_t         counter_set_id[0x8];
     uint8_t         uar_page[0x18];
 
-    uint8_t         reserved_at_80[0x8];
+    uint8_t         reserved_at_80[0x3];
+    uint8_t         full_handshake[0x1];
```

Comment:
minor: why used uint8[1] instead of just uint8 value?

Reply:
I guess, It is aligned with PRM definition